tensorflow/compiler/mlir/tools/kernel_gen/transforms/tf_to_jit_invocations.cc

```diff
@@ -115,20 +119,33 @@ struct TFToI64JITInvocationForLargeTensorsPattern : public RewritePattern {
 
   LogicalResult matchAndRewrite(Operation *op,
```

Comment:
Can we  add a test for this case in `compiler/mlir/tools/kernel_gen/tests/tf_to_jit_invocations.mlir`?

Reply:
added.